Gems/PhysX/Code/Source/Articulation.cpp

```diff
@@ -62,50 +62,44 @@ namespace PhysX
 
     void ArticulationLink::AddCollisionShape(const ArticulationLinkData& thisLinkData)
     {
-        const Physics::ColliderConfiguration* colliderConfiguration = thisLinkData.m_shapeColliderConfiguration.first.get();
-        const Physics::ShapeConfiguration* shapeConfiguration = thisLinkData.m_shapeColliderConfiguration.second.get();
-
-        if (shapeConfiguration && colliderConfiguration)
+        m_physicsShapes.clear();
+        for (auto &[colliderConfiguration, shapeConfiguration]: thisLinkData.m_shapeColliderConfigurationList)
```

Comment:
nitpick: the & should be moved to the attached to the "auto" keyword
```suggestion
        for (auto& [colliderConfiguration, shapeConfiguration] : thisLinkData.m_shapeColliderConfigurationList)
```